zerver/webhooks/clickup/__init__.py

```diff
@@ -0,0 +1,39 @@
+from enum import Enum
+from typing import List
+
+
+class ConstantVariable(Enum):
+    @classmethod
+    def as_list(cls) -> List[str]:
+        return [item.value for item in cls]
+
+
+class EventItemType(ConstantVariable):
```

Comment:
Constant variables are declared in `clickup/__init__.py` and not in `view.py` to avoid circular import because `api_endpoints.py` also uses a common constant variable; `EventItemType`.